test/services/dumpers/agent-nodejs/agent-nodejs.test.ts

```diff
@@ -0,0 +1,478 @@
+          cleanStub(osStub);
+        });
+
+        describe('when the database is local on the system', () => {
+          it('should not use `network` option set to `host` in the docker-compose file', async () => {
+            expect.assertions(1);
+
+            const { osStub, postgresConfig } = await setupAndDump();
+            const dockerComposeFile = fs.readFileSync(
+              `${appRoot}/${postgresConfig.appConfig.applicationName}/docker-compose.yml`,
+              'utf-8',
+            );
+            expect(dockerComposeFile).not.toContain('network');
+
+            rimraf.sync(`${appRoot}/${postgresConfig.appConfig.applicationName}`);
+            cleanStub(osStub);
+          });
+        });
+
+        describe('when the database is not local on the system', () => {
+          it('should not use `network` option in the docker-compose file', async () => {
+            expect.assertions(1);
+
+            const { osStub, postgresConfig } = await setupAndDump('mysql://example.com:8999');
+
+            const dockerComposeFile = fs.readFileSync(
+              `${appRoot}/${postgresConfig.appConfig.applicationName}/docker-compose.yml`,
+              'utf-8',
+            );
+            expect(dockerComposeFile).not.toContain('network');
+
+            rimraf.sync(`${appRoot}/${postgresConfig.appConfig.applicationName}`);
+            cleanStub(osStub);
+          });
+        });
+      });
+    });
+
+    describe('with a FOREST_SERVER_URL', () => {
+      const setupAndDump = async (dbConnectionUrl: string | null = null) => {
+        const postgresConfig = {
+          appConfig: {
+            applicationName: 'test-output/postgres',
+            appHostname: 'localhost',
+            appPort: 1654,
+            path: appRoot,
+          },
+          dbConfig: {
+            dbDialect: 'postgres',
+            dbConnectionUrl: 'postgres://localhost:54369',
+            ssl: false,
+            dbSchema: 'public',
+          },
+          forestAuthSecret: 'forestAuthSecret',
+          forestEnvSecret: 'forestEnvSecret',
+        };
+        const injectedContext: any = Context.execute(defaultPlan);
+        const dumper = new AgentNodeJs({
+          ...injectedContext,
+          env: {
+            ...injectedContext.env,
+            FOREST_SERVER_URL: 'http://localhost:3001',
+            FOREST_URL_IS_DEFAULT: false,
+          },
+        });
+        await dumper.dump(
+          {
+            appConfig: postgresConfig.appConfig,
+            dbConfig: {
+              ...postgresConfig.dbConfig,
+              dbConnectionUrl: dbConnectionUrl || postgresConfig.dbConfig.dbConnectionUrl,
+            },
+            forestAuthSecret: 'forestAuthSecret',
+            forestEnvSecret: 'forestEnvSecret',
+          },
+          {},
+        );
+
+        return postgresConfig;
+      };
+
+      it('should set `forestServerURl` in `.env`, `docker-compose.yml` and `index.js`', async () => {
```

Comment:
When something is tests, the opposite should always be tested (even if it is Intrinsically tested by some other tests). 

Describe should be pretty clear about what is expected.

Reply:
👍